<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemManagement</class>
 <widget class="QWidget" name="SystemManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统管理</string>
  </property>
  <widget class="QPushButton" name="saveDataButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="mainTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="studentTab">
    <attribute name="title">
     <string>学生管理</string>
    </attribute>
    <widget class="QTableView" name="studentTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>371</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="staffTab">
    <attribute name="title">
     <string>教师管理</string>
    </attribute>
    <widget class="QTableView" name="staffTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>391</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="courseTab">
    <attribute name="title">
     <string>课程管理</string>
    </attribute>
    <widget class="QTableView" name="courseTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>371</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="teacherRankTab">
    <attribute name="title">
     <string>教学效果</string>
    </attribute>
    <widget class="QTableView" name="teacherRankTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>371</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="removeRowButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除当前行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addRowButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加新行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
